edb/server/args.py

```diff
@@ -316,6 +320,22 @@ def _writer(status: str) -> None:
         '--auto-shutdown-after', type=float, default=-1.0,
         help='shutdown the server after the last connection has been closed '
              'for N seconds. N < 0 is treated as infinite.'),
+    click.option(
+        '--tls-cert-file', type=PathPath(),
+        help='Specify a path to a single file in PEM format containing the TLS'
+             'certificate as well as any number of CA certificates needed to '
+             'establish the certificate’s authenticity. If not present, a '
```

Comment:
I'd do this differently.

`edb server` should have `--tls-cert-file` optional.
`edgedb-server` should have it required.

With adjusted help section for each tool.

Reply:
Had to add a hidden `edb testserver` sub-command for the tests to leverage the cert auto-generation feature, at least the tests passed now.

Reply:
What is the reason for avoiding generating dev certs in `edb server`?

Reply:
Oh we also generate dev certs in `edb server`, it's just that neither `edb server` nor `edgedb-server` (`python -m edgedb.server.main`) is suitable for running the tests. `edb server` has `insecure=True` and `edgedb-server` doesn't generate the certs.